lib/ansiblelint/errors.py

```diff
@@ -19,19 +19,35 @@ def __repr__(self):
                                 self.filename, self.linenumber, self.line)
 
 
+class AnsibleLintInternalRule():
+    """Internal error likely cause by parsing failures."""
+
+    id = "999"
+    shortdesc = "Parsing Error"
+    description = "Ansible failed to parse that file."
```

Comment:
'Failed to parse file.' is good enough.
Anyway it's not ansible that fails to parse the fail it's ansible-lint that errors.